.prettierrc

```diff
@@ -2,5 +2,8 @@
   "trailingComma": "all",
   "tabWidth": 2,
   "semi": true,
-  "singleQuote": true
+  "singleQuote": true,
+  "importOrder": ["^@?graasp", "^@?material", "^[./]"],
```

Comment:
Would it make sense to have material before graasp? So that we have imports from least to most specific
Do we need a react group?